js/ui/scheduler/ui.scheduler.appointments.strategy.base.js

```diff
@@ -519,10 +523,11 @@ var BaseRenderingStrategy = Class.inherit({
 
     getAppointmentDataCalculator: noop,
 
-    _customizeCoordinates: function(coordinates, ratio, appointmentCountPerCell, maxHeight, isAllDay) {
+    _customizeCoordinates: function(coordinates, ratio, appointmentCountPerCell, topOffset, maxHeight, isAllDay) {
```

Comment:
A lot of arguments...

`ratio` and `maxHeight` are used only for `height` calculation. Can we calculate `height` outside and pass it to `_customizeCoordinates`?

The same about `topOffset`.